skstan/utils/functions.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+
+def sigmoid_each(x: np.array):
+    return 1 / (1 + np.exp(-x))
```

Comment:
Write `1.0` in advance. At this line, int type is  casted to float type implicitly.